### PR TITLE
MODCFIELDS-2: Add endpoints to raml file and ModuleDescriptor

### PR DESCRIPTION
**Pull request by andrii-paias**
## Purpose
Add unimplemented endpoints for custom fields

## Approach
Declare endpoints for basic CRUD operations for custom fields
Add schemas and examples
Add permissions

### CODE REVIEW

**Review by natali-zaitseva — APPROVED** [review]
thanks for adding a Pull request template!

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -11,6 +11,26 @@
           "methods": ["GET"],
           "pathPattern": "/custom-fields",
           "permissionsRequired": ["custom.fields.collection.get"]
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/custom-fields",
+          "permissionsRequired": ["custom.fields.item.post"]
```
Comment (sduvvuri-ebsco) [inline descriptors/ModuleDescriptor-template.json:18]: Shouldn't this be collection.post?

File: ramls/customField.json [hunk 2]
```diff
@@ -18,7 +12,7 @@
     "refId": {
       "type": "string",
       "description": "The reference id of the custom field",
-      "example": "packages"
+      "example": "department"
```
Comment (sduvvuri-ebsco) [inline ramls/customField.json:15]: is this a valid example for a refID?

**Review by sduvvuri-ebsco — COMMENTED** [review]
I know this PR is merged but had a few questions/comments @andrii-paias . Thanks!

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (andrii-paias) [reply descriptors/ModuleDescriptor-template.json:18]: it seems to be common to have permission named item.post
https://github.com/search?q=org%3Afolio-org+item.post+filename%3AModuleDescriptor-template.json&type=Code

**Pull request merged**